README.md

```diff
@@ -63,6 +63,8 @@ Deployments are done using GitHub Actions. To deploy a branch, follow the follow
 - Choose a branch and if you want to merge the changes on the staging branch into the master branch (only possible when the branch chosen in previous step is master).
 - Click the green button "Run workflow".
 
+*Note: If the workflow fails with an error message about blobs along the lines of the following: `buildx failed with: ERROR: failed to solve: [...] blob [...]`: remove all recent cache files for staging on the [Caches page](https://github.com/csvalpha/amber-ui/actions/caches) and try again. You might have to turn on the option `Perform all jobs, regardless of whether there are actual changes?` to force the workflow to re-execute all its jobs.*
```

Comment:
Odd bug, could this be a one-time bug caused by https://github.blog/changelog/2023-12-14-github-actions-artifacts-v4-is-now-generally-available/? That is artifacts and not cache though, but it might be related.

Reply:
resolving because I don't know what to do with this and the remark does not seem prohibitive of merging this PR